common/makefiles/generic-make-go.mk

```diff
@@ -0,0 +1,249 @@
+# Default configuration
```

Comment:
Let's get rid of this immediately. As far as I could see in test-infra, we only use the `lint` and `test-local` targets from our own Makefile. Let's revert this to something close to a "normal" Kubebuilder project Makefile.